config/environments/production.rb

```diff
@@ -26,9 +26,13 @@
   # Use a different logger for distributed setups
   # config.logger = SyslogLogger.new
 
-  config.lograge.enabled = true
-  config.log_level = :error
-  config.lograge.formatter = Lograge::Formatters::Logstash.new
+  config.logstash.progname = 'Jobsworth'
+  config.log_level = :info
+  config.logstash.formatter = :json_lines
+  config.logstash.port = config.try(:jobsworth).try(:logstash_port) || '5000'
+  config.logstash.type = :tcp
+  config.logstash.host = config.try(:jobsworth).try(:logstash_host) || '203.29.62.201'
```

Comment:
Let's not put this IP in the public repo. It should have no fallback at all. If not configured, then it should disable itself.